### DISCUSSION TITLE
Phase 2 features

### DISCUSSION THREAD

**Discussion by mbradds** [mbradds/pipeline-profiles#94] [Ideas]
Most of the new and redesigned sections of the pipeline profiles have a completed and released first version. This is an area to keep track of additional features and ideas that can improve these first versions.

## General
- Look into using percentages for height instead of media queries.
- Add a reset map button to the leaflet maps, and remove the "refill map bubbles" button from the near me section. This will make these dashboards make more sense and improve user understanding.
- Add a fullscreen option for some, or all of the dashboards.

## Contaminated Sites & Remediation
- The open gov data contains some cool fields that I wasn't aware of. It might be cool to add a click pop-up on each map site to show things like the "Annual Update Description Of Changes", "Annual Update Summary Of Activities", "Contaminated Site Description", "Summary of Actions Planned to Mitigate Contamination", "Summary of Actions Taken to Mitigate Contamination". This could probably be a leaflet pop-up following the same pattern as the IAMC prototype.

### O&M Activities
- Add dynamic text about O&M expenditures/investment. Or just a link to QSR filings.

## Incidents
- The sidebar buttons could be improved.

**Comment by ghost** [comment]
Comments from engagement that could be improvements in v.2. 

- Can I download a map of the contaminated site along with the details about it, so I can refer to it offline?
- Can I link from the site on the map directly to an incident report? Or inspection reports if done for that site or area?
